src/options/view.rs

```diff
@@ -257,6 +257,9 @@ impl TimeFormat {
         if &word == "default" {
             Ok(Self::DefaultFormat)
         }
+        else if &word == "relative" {
+            Ok(Self::Relative)
+        }
         else if &word == "iso" {
             Ok(Self::ISOFormat)
         }
```

Comment:
Let’s use a match here.

Reply:
I updated the code, this needs a conversion to `str` to look nice-ish.